### ISSUE TITLE
get_current_game_info_by_summoner panics in custom games.

### ISSUE THREAD

**Issue by Scarjit** [MingweiSamuel/Riven#2]
### Description
When using the `spectator_v4.get_current_game_info_by_summoner()` API, to lookup a summoner, who is currently playing a Custom Game, it will panic, due to missing `gameQueueConfigId`.

## Steps to reproduce

1. Join a custom game
2. Lookup the encrypted summoner id & correct region, with `spectator_v4.get_current_game_info_by_summoner()`.
3. Riven should panic.

## Expected outcome
Correct parsing of API response.
Example response:
```
{
    "gameId": 4302399720,
    "gameStartTime": 1575296347917,
    "platformId": "EUW1",
    "gameMode": "CLASSIC",
    "mapId": 11,
    "gameType": "CUSTOM_GAME",
    "bannedChampions": [],
    "observers": {
        "encryptionKey": "rvOzZDwZdkQg8ET3Nqr4/fHJ67vbuIBO"
    },
    "participants": [
        {
            "profileIconId": 3545,
            "championId": 429,
            "summonerName": "S1mplescripts",
            "gameCustomizationObjects": [],
            "bot": false,
            "perks": {
                "perkStyle": 8200,
                "perkIds": [
                    8214,
                    8226,
                    8233,
                    8237,
                    8313,
                    8321,
                    5008,
                    5008,
                    5002
                ],
                "perkSubStyle": 8300
            },
            "spell2Id": 4,
            "teamId": 100,
            "spell1Id": 7,
            "summonerId": "frGEWr2kt1fizZRVHY9IuJLEG7LDfVGjVE0BYT0MB-oJK3Du"
        }
    ],
    "gameLength": 87
}
```

**Comment by MingweiSamuel** [comment]
Perfect, thank you for the report

**Comment by MingweiSamuel** [comment]
Should be fixed in version `0.1.3`. Reopen if not fixed

https://github.com/MingweiSamuel/riotapi-schema/commit/699ecd182c5f6a9e077e195541510d44254e6b15
https://github.com/MingweiSamuel/Riven/commit/2233be4dcbd201386f77a661161c8cc6a93a3450#diff-073559fc80dd0ef36481dadf680965e3L962
